src/main/java/rx/internal/schedulers/NewThreadWorker.java

```diff
@@ -30,24 +34,111 @@
     private final ScheduledExecutorService executor;
     private final RxJavaSchedulersHook schedulersHook;
     volatile boolean isUnsubscribed;
-
+    /** The purge frequency in milliseconds. */
+    private static final String FREQUENCY_KEY = "io.reactivex.rxjava.scheduler.jdk6.purge-frequency-millis";
```

Comment:
These don't match the naming convention using in `RxRingBuffer` with just the `rx` prefix: https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/internal/util/RxRingBuffer.java#L267

We should probably stick with that convention since it is already set, so:

`rx.scheduler.jdk6.purge-frequency-millis`
`rx.scheduler.jdk6.purge-force`